Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.MeshSync.Editor
+{
+    internal class BlenderLauncher : IDCCLauncher
+    {
+        const string menuItem_OpenInBlender = "Assets/MeshSync/Open in blender";
+
+        const string editorSettingPath = "MESHSYNC_BLENDER_PATH";
+
+        System.Diagnostics.Process m_blenderProcess;
+        static bool redirectBlenderToUnityConsole;
+
+        public RunMode runMode { get; set; }
+
+        ~BlenderLauncher()
+        {
+            Cleanup();
+        }
+
+        static string GetBlenderPath()
+        {
+            var blenderPath = EditorPrefs.GetString(editorSettingPath);
+
+            if (string.IsNullOrEmpty(blenderPath))
+            {
+                if (!DefaultAppUtility.TryGetRegisteredApplication(".blend", out blenderPath))
+                {
+                    if (Application.platform == RuntimePlatform.OSXEditor)
+                    {
+                        blenderPath = "/Applications/Blender.app/Contents/MacOS/Blender";
+                    }
+                }
+
+                SetBlenderPath(blenderPath);
+            }
+
+            return blenderPath;
+        }
+
+        public static void SetBlenderPath(string blenderPath)
+        {
+            EditorPrefs.SetString(editorSettingPath, blenderPath);
+        }
+
+        [MenuItem(menuItem_OpenInBlender, priority = 0)]
+        static void OpenBlender()
+        {
+            var selectedAsset = Selection.objects[0];
+            if (selectedAsset == null)
+                return;
+
+            var servers = UnityEngine.Object.FindObjectsOfType<MeshSyncServer>();
+
+            MeshSyncServer server = null;
+
+            foreach (var serverInScene in servers)
+            {
+                if (serverInScene.enabled)
+                {
+                    server = serverInScene;
+                    break;
+                }
+            }
+
+            if (server == null)
+            {
+                server = MeshSyncMenu.CreateMeshSyncServerMenu(null);
```

Comment:
If there is no server in the scene and the user right clicks a blend file in the asset library to open it in blender, this creates a server in the scene so that the realtime updates work. Otherwise the user could just double click the blend file to open it without meshsync.